.github/workflows/validate.yml

```diff
@@ -75,9 +75,9 @@ jobs:
         run: npm run prettier-check:updated
       - name: Validate ESLint rules
         run: npm run lint:updated
-      - name: Validate commit messages
-        if: github.event.pull_request.base.repo.id == github.event.pull_request.head.repo.id
-        run: npx commitlint -f master
+      # - name: Validate commit messages
```

Comment:
I believe this should not be here

Reply:
Oops!  Thanks for catching.  I reverted it.